Fixes and improvements

eltimen opened the pull request.

eltimen committed:
fix incomplete child elements for some windows

eltimen committed:
rename exceptions and fix PEP-8 warnings

eltimen committed:
update binaries

vasily-v-ryabov reviewed (commented):
Looks good to me. It would be nice to fix 1 Codacy issue.

eltimen committed:
rename class Main to Server

eltimen committed:
add GetProperties action

eltimen committed:
update binaries

eltimen committed:
get CLR version from EnumerateLoadedRuntimes or EnumerateInstalledRuntimes

eltimen committed:
cleanup logs from bootstrap.dll

eltimen committed:
fix some codacy warnings

eltimen committed:
remove unused type parameters from base action classes

eltimen committed:
add ability to disconnect and shutdown server

not used from client side yet
unloads bootstrap.dll, but it's need to implement unloading of managed assemblies
pipeline "start() -> manually send shutdown command -> connect()" works fine

eltimen committed:
add tree_mode parameter to GetChildren action

can be useful in case of incorrect extraction of children of some custom control
values: visual, logical, visual_custom
visual_custom is default mode based on visual tree, but uses some overloads for controls derived from specified classes

eltimen committed:
update binaries

eltimen committed:
fix empty children of some ContentControl-derived elements

eltimen committed:
update binaries
